uri-schemes.md

```diff
@@ -0,0 +1,230 @@
+# Deltachat URI-Schemes
+
+Currently URI schemes are mainly supported by the app-internal QRcode scanner, at this time only `openpgp4fpr` (used for securejoin) is also registered as uri scheme handler on all platforms ([android](https://github.com/deltachat/deltachat-android), [ios](https://github.com/deltachat/deltachat-ios) and [desktop](https://github.com/deltachat/deltachat-desktop))
+
+- related forum topic: https://support.delta.chat/t/custom-deltachat-url-uri-scheme/346
+- core sourcecode that handles uri schemes: https://github.com/deltachat/deltachat-core-rust/blob/master/src/qr.rs
+
+## **chat.delta**
+
+Generic deltachat scheme, currently used by android as callback url for the oauth2 login method.
+| | |
+| ------------------- | ------------------------------- |
+| Scheme | `chat.delta` |
+| Used for | oauth2 Login (only android currently) |
+| Related Terms\* | - |
+| Availible on | iOS (defined, but not used yet), android |
+| Handled by the core| No|
+| Other apps using it | none, only DeltaChat |
+| Notes | Only used as scheme currently, not in qr codes |
+
+### Syntax
+
+[TODO]
+
+### Reference
+
+https://github.com/deltachat/deltachat-android/blob/8d9c02dd7a2be33fe588e0b9469be56fb0435a62/AndroidManifest.xml#L192-L195
+
+```xml
+<!-- this scheme is used as the redirect_url for getOauth2Url()
+    and should be whitelisted by the supported oauth2 services -->
+<data android:scheme="chat.delta" android:path="/${applicationId}/auth" tools:ignore="AppLinkUrlError"/>
+<data android:scheme="chat.delta" android:path="/auth" tools:ignore="AppLinkUrlError"/>
+```
+
+## **OPENPGP4FPR** <a name="OPENPGP4FPR"></a>
+
+|                         |                                 |
+| ----------------------- | ------------------------------- |
+| Scheme                  | `OPENPGP4FPR:`                  |
+| Used for                | Verify Contact and Join a group |
+| Related Terms\*         | Securejoin                      |
+| Availible on            | iOS, android, desktop           |
+| Decoded by the core\*\* | Yes                             |
+| Other apps using it     | <li>OpenKeychain(android)</li>  |
+
+### Syntax
+
+```
+OPENPGP4FPR:FINGERPRINT#a=ADDR&n=NAME&i=INVITENUMBER&s=AUTH
+OPENPGP4FPR:FINGERPRINT#a=ADDR&g=GROUPNAME&x=GROUPID&i=INVITENUMBER&s=AUTH
```

Comment:
a, g and n are URL-encoded, not sure if you need to point this out.
i & s are 66 random bits endcoded as base-64.